data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/past_usual_household_address.jsonnet

```diff
@@ -81,6 +81,18 @@ local proxyDescription = 'If they had no usual address one year ago, state the a
         ],
       },
     },
+    {
+      goto: {
+        block: 'last-year-address',
+        when: [
+          {
+            id: 'past-usual-address-household-answer',
+            condition: 'equals',
+            value: 'Student term time address in the UK',
```

Comment:
You could remove this extra block and use the `equals any` rule.
```jsonnet
{
      goto: {
        block: 'last-year-address',
        when: [
          {
            id: 'past-usual-address-household-answer',
            condition: 'equals any',
            values: [
              'Another address in the UK',
              'Student term time address in the UK',
            ],
          },
        ],
      },
    },
```

Reply:
Another good idea, I didn't know about equals any

Reply:
done